GPT/gpt.py

```diff
@@ -118,6 +118,12 @@ def paste_and_select(result: str):
         for _ in result:
             actions.edit.extend_left()
 
+    def paste_and_append(result: str):
```

Comment:
Honestly not sure if we need this function. I think it makes sense to keep stuff in the `.talon` file if it is just talon actions without any Python code. I like to limit stuff in the global namespace if possible. Is that reasonable?

Reply:
That makes sense to me! ✅